pages/Admins/index.tsx

```diff
@@ -1,8 +1,28 @@
+import * as React from 'react';
 import Layout from "../../components/Layout/Layout";
+import styles from './Admins.module.css';
+import { Grid, Button, Box } from "@mui/material";
 
 const AdminPage: React.FunctionComponent = () => {
+
     return(
-        <Layout title='Admins'/>
+      <main>
+        <Layout title='Admins'>
+          <main className={styles.main}>
+            <Grid container className={styles.container}>
+              <Grid item xs={8}>
+                <h2 className={styles.h2}>ADMIN ACCOUNTS</h2>
+              </Grid>
+              <Grid item xs={2}>
+                <Button className={styles.button}>INVITE ADMIN</Button>
```

Comment:
I think the overall positioning is a bit off from the Figma designs (it's supposed to be positioned all the way to the right).